### DISCUSSION TITLE
雑談

### DISCUSSION THREAD

**Discussion by Aiueokashi** [Project-DUCK/main-code#19] [General]
雑談用

**Comment by Aiueokashi** [comment]
あいうえお菓子

**Reply by Aiueokashi** [reply]
Reply

**Reply by pippi0057** [reply]
れply

**Comment by pippi0057** [comment]
new

**Comment by Aiueokashi** [comment]
A

**Comment by Aiueokashi** [comment]
お話しできるぞやったね

**Comment by pippi0057** [comment]
main codeで雑談草

**Comment by Aiueokashi** [comment]
- [ ] マークダウン

# マークダウン

**Comment by pippi0057** [comment]
privateでできたら完璧だった

**Comment by Aiueokashi** [comment]
![5A555CC8-9DC0-4290-B775-4A99EEE3D01A](https://user-images.githubusercontent.com/70052601/106688548-aa259900-6611-11eb-87df-6dac44fd0ee3.png)

写真もいける！

**Comment by Aiueokashi** [comment]
@pippi-sniper 

え、てか他人のメッセージ編集できるのやばすぎない？wwwww

**Reply by pippi0057** [reply]
なんかメンションされてた

**Comment by pippi0057** [comment]
真面目な目的でも使えそう(そっちがgithubの想定してる使い方)

**Comment by Aiueokashi** [comment]
![badge](https://img.shields.io/badge/DUCK-DUCK-green)

**Comment by pippi0057** [comment]
OwO

**Comment by Drkoukichi** [comment]
otintin

**Reply by Aiueokashi** [reply]
全githubユーザーが見れるけど大丈夫化か？w

**Reply by pippi0057** [reply]
何も大丈夫じゃなくて草

**Reply by Aiueokashi** [reply]
ぽまえタブレット持ってるやん

**Reply by pippi0057** [reply]
家に置いてきたと思ってたけど鞄の中に入ってた
